packages/wonder-blocks-data/src/util/response-cache.js

```diff
@@ -29,32 +28,35 @@ export class ResponseCache {
         return _default;
     }
 
-    _hydrationCache: MemoryCache<any, any>;
-    _ssrOnlyCache: ?MemoryCache<any, any>;
+    _hydrationCache: MemoryCache;
+    _ssrOnlyCache: ?MemoryCache;
 
     constructor(
-        hydrationCache: ?MemoryCache<any, any> = null,
-        ssrOnlyCache: ?MemoryCache<any, any> = null,
+        hydrationCache: ?MemoryCache = null,
+        ssrOnlyCache: ?MemoryCache = null,
     ) {
         this._ssrOnlyCache = Server.isServerSide()
             ? ssrOnlyCache || new MemoryCache()
             : undefined;
         this._hydrationCache = hydrationCache || new MemoryCache();
     }
 
-    _setCacheEntry<TOptions, TData: ValidData>(
-        handler: IRequestHandler<TOptions, TData>,
-        options: TOptions,
+    _setCacheEntry<TData: ValidData>(
+        id: string,
         entry: CacheEntry<TData>,
+        hydrate: boolean,
     ): CacheEntry<TData> {
         const frozenEntry = Object.freeze(entry);
-        if (this._ssrOnlyCache != null) {
+        if (Server.isServerSide()) {
```

Comment:
I did this just so that the TrackData docs example would still work. I didn't like it, but it seemed OK for now. better docs will be coming that make it less ...flaky (the docs are flaky, not the code)?